pkgs/development/python-modules/okonomiyaki/default.nix

```diff
@@ -0,0 +1,43 @@
+{ buildPythonPackage
+, fetchFromGitHub
+, lib
+, attrs
+, distro
+, jsonschema
+, six
+, zipfile2
+, hypothesis
+, mock
+, packaging
+, testfixtures
+}:
+
+buildPythonPackage rec {
+  pname = "okonomiyaki";
+  version = "1.3.2";
+
+  src = fetchFromGitHub {
+    owner = "enthought";
+    repo = pname;
+    rev = version;
+    hash = "sha256-eWCOuGtdjBGThAyu15aerclkSWC593VGDPHJ98l30iY=";
+  };
+
+  propagatedBuildInputs = [ distro attrs jsonschema six zipfile2 ];
+
+  postPatch = ''
+    substituteInPlace okonomiyaki/runtimes/tests/test_runtime.py \
+      --replace 'runtime_info = PythonRuntime.from_running_python()' 'raise unittest.SkipTest() #'
+  '';
+
+  checkInputs = [ hypothesis mock packaging testfixtures ];
```

Comment:
Not sure how to disable those tests for darwin

```
FAIL: test_platform_tag (okonomiyaki.platforms.tests.test_pep425.TestPEP425)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/private/tmp/nix-build-python3.10-okonomiyaki-1.3.2.drv-0/source/okonomiyaki/platforms/tests/test_pep425.py", line 70, in test_platform_tag
    self.assertEqual(platform_tag, self.tag.platform)
AssertionError: 'macosx_10_16_x86_64' != 'macosx_13_0_x86_64'
- macosx_10_16_x86_64
?          ---
+ macosx_13_0_x86_64
?         ++
======================================================================
FAIL: test_platform_tag_default (okonomiyaki.platforms.tests.test_pep425.TestPEP425)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/private/tmp/nix-build-python3.10-okonomiyaki-1.3.2.drv-0/source/okonomiyaki/platforms/tests/test_pep425.py", line 77, in test_platform_tag_default
    self.assertEqual(platform_tag, self.tag.platform)
AssertionError: 'macosx_10_16_x86_64' != 'macosx_13_0_x86_64'
- macosx_10_16_x86_64
?          ---
+ macosx_13_0_x86_64
?         ++
```

Reply:
an `optionalString` appended to `postPatch`?

Reply:
It should probably be done in preCheck and if there is no argument that should work.

Reply:
done